client/js/components/map/publicdetail/controls/layerlist/DpPublicLayerList.vue

```diff
@@ -23,7 +23,7 @@
         :layer-groups-alternate-visibility="layerGroupsAlternateVisibility" />
       <dp-public-layer-list-category
         v-else
-        :key="layer.id"
+        :key="`category:${layer.id}?`"
```

Comment:
nitpick: do we really need the question mark as a string here?

Reply:
no. That makes no sense
```suggestion
        :key="`category:${layer.id}`"
```